README.md

```diff
@@ -1,2 +1,88 @@
-# icepy
-Python-esque programming language with Icelandic grammar rules
+# IcePy
+
+A programming language that has strict Icelandic grammar rules built in.
+
+## Concepts
+
+There are a set of known variable names that can be used. Each have forms in
+nominative, accusative, dative, genitive.
+
+Each argument to a function must define the case expected.
+
+Each class has a gender
+
+Core constructs of the language may take different cases too.
+
+
+## Language constructs
+
+As the name suggests, the language is python-esque. Except that there are mass
+translations of things (eg `class` and `def` are no longer available).
+
+
+### Defining a method or function
+
+To define the function "reikna", you use the keyword "virka"
+
+```
+virka reikna():
+    pass
+```
+
+### Defining a class (tegund)
+
+```
+tegund Bátur():
+
+   einstaklingar = []
+
+   virka __frumstilla__(sjalfur):
+       pass
```

Comment:
`sjálft` might be a better generic, even though Icelandic does have a tendency to default to masculine.

Perhaps `ekkert` or `tómt` for `pass` - since pass is essentially a no-op?

`skila` for return - maybe a bit literal?